script/output/1/base-release-20241017.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
This is not actually released on mainnet and shouldn't be included in the PR. The actual deployment should be added.

Reply:
Removed